cmake/Dependencies.cmake

```diff
@@ -43,9 +43,14 @@ if(NOT APPLE)
     find_package(PNG REQUIRED)
     include_directories(BEFORE SYSTEM ${PNG_PNG_INCLUDE_DIR})
 	# Find Python Libraries
-	find_package(PythonLibs)
+	find_package(PythonLibs 3.5)
 endif()
 
+find_program(PYSIDE_UIC NAME pyside-uic
+		HINTS "${PYTHON_INCLUDE_DIRS}/../Scripts"
```

Comment:
Will that hint work for all platforms?

Reply:
Yup, pyside-uic is in system path on linux by default.